### PR TITLE
Userguide: add a visualization chapter

### PR DESCRIPTION
**Pull request by sciunto**
## Description

Visualuzation tools are missing in our userguide. I started a simple chapter. I'm not enthusiast to write pages of documentation and I believe that listing the libraries already makes 80% of the job (pareto... :) ).

If you want to see and write more, please, do not hesitate to submit a PR.

I think this would interest @emmanuelle  and @jni 

Note that I flipped the position of two chapters. In my head data format, IO, visualization are one part, and image processing another one. I was confused by the former progression order, but it could be a matter of taste.

## Checklist

- [ ] [Docstrings for all functions](https://github.com/numpy/numpy/blob/master/doc/example.py)
- [ ] Gallery example in `./doc/examples` (new features only)
- [ ] Benchmark in `./benchmarks`, if your changes aren't covered by an
  existing benchmark
- [ ] Unit tests
- [ ] Clean style in [the spirit of PEP8](https://www.python.org/dev/peps/pep-0008/)

## For reviewers

- [ ] Check that the PR title is short, concise, and will make sense 1 year
  later.
- [ ] Check that new functions are imported in corresponding `__init__.py`.
- [ ] Check that new features, API changes, and deprecations are mentioned in
      `doc/release/release_dev.rst`.

### CODE REVIEW

File: doc/source/user_guide/visualization.rst [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+------------------
+
+Data visualization takes an important part in image processing. Data can be
+a simple unique 2D image or a more complex with multidimensional aspects: 3D
+in space, timeslapse, multiple channels.
+
+Therefore, the visualization strategy will depend on the data complexity and
+a range of tools external to scikit-image can be used for this purpose.
+Historically, scikit-image provided viewer tools but new powerful softwares
+are now developed and must be preferred.
```
Comment (emmanuelle) [inline doc/source/user_guide/visualization.rst:11]:
```suggestion
are now available and must be preferred.
```

File: doc/source/user_guide/visualization.rst [hunk 2]
```diff
@@ -0,0 +1,37 @@
+Data visualization
+------------------
+
+Data visualization takes an important part in image processing. Data can be
```
Comment (emmanuelle) [inline doc/source/user_guide/visualization.rst:4]:
```suggestion
Data visualization takes an important place in image processing. Data can be
```

File: doc/source/user_guide/visualization.rst [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+a simple unique 2D image or a more complex with multidimensional aspects: 3D
+in space, timeslapse, multiple channels.
+
+Therefore, the visualization strategy will depend on the data complexity and
+a range of tools external to scikit-image can be used for this purpose.
+Historically, scikit-image provided viewer tools but new powerful softwares
```
Comment (emmanuelle) [inline doc/source/user_guide/visualization.rst:10]:
```suggestion
Historically, scikit-image provided viewer tools but powerful packages
```

File: doc/source/user_guide/visualization.rst (hunk 1 above)
Comment (emmanuelle) [reply doc/source/user_guide/visualization.rst:11]: some of these packages have been around the block for more than a decade so I would not insist too much on the new/developed :-).

Comment (sciunto) [reply doc/source/user_guide/visualization.rst:11]: haha You're right :) I had only napari in my buffer.

**Comment by rfezzani** [comment]
Thank you @sciunto ;)

**Pull request merged**